pkgs/applications/networking/nordvpn/default.nix

```diff
@@ -0,0 +1,82 @@
+{ autoPatchelfHook, buildFHSEnv, dpkg, fetchurl, lib, stdenv, sysctl
+, iptables, iproute2, procps, cacert, libxml2, libidn2, zlib, wireguard-tools }:
+
+let
+  pname = "nordvpn";
+  version = "3.15.5";
+
+  nordVPNBase = stdenv.mkDerivation {
+    inherit pname version;
+
+    src = fetchurl {
+      url =
+        "https://repo.nordvpn.com/deb/nordvpn/debian/pool/main/nordvpn_${version}_amd64.deb";
+      hash = "sha256-7IvtpWiwVFaOc9/2NzG9mAGaWAeBZXNPdqCumdslawI=";
+    };
+
+    buildInputs = [ libxml2 libidn2 ];
+    nativeBuildInputs = [ dpkg autoPatchelfHook stdenv.cc.cc.lib ];
+
+    dontConfigure = true;
+    dontBuild = true;
+
+    unpackPhase = ''
+      runHook preUnpack
+      dpkg --fsys-tarfile $src | tar --extract
+      runHook postUnpack
+    '';
+
+    installPhase = ''
+      runHook preInstall
+      mv usr/ $out/
+      mv var/ $out/
```

Comment:
If it's just connection profiles in `$out/var`, put them into `$out/etc`.

Reply:
there is a bunch of files in `/var/lib/nordvpn` including a public key, which are assumed by the daemon to be in this folder.